distrho/src/DistrhoPluginVST3.cpp

```diff
@@ -359,14 +359,14 @@ class PluginVst3
                     midiEvent.size = 3;
                     midiEvent.data[0] = 0x90 | (eventStorage.noteOn.channel & 0xf);
                     midiEvent.data[1] = eventStorage.noteOn.pitch;
-                    midiEvent.data[2] = std::max(0, std::min(127, (int)(eventStorage.noteOn.velocity * 127)));
+                    midiEvent.data[2] = std::max(0, std::min(127, (int)(eventStorage.noteOn.velocity / 0.0078125)));
```

Comment:
dont use magic values please, why is that better than * 127 ?

Reply:
because for a CC value of 1, normalized is 0.007874 and `(int)(0.007874 * 127)` is 0, it's just a rounding issue so `std::round` works too and is indeed better.  
`(int)std::round(eventStorage.noteOn.velocity * 127)`